design/010-new-dataset-fields/010-new-dataset-fields.md

```diff
@@ -0,0 +1,130 @@
+* **Value:** [URL](https://schema.org/URL) from https://ror.org/
+* **Example:**
+```json
+"http://schema.org/affiliation": [
+    {
+    "@type": "Organization",
+    "name": "EPFL",
+    "identifier": "https://ror.org/02s376052"
+    }
+```
+
+> Note: ROR has a public REST API which could be used to autosuggest records in a text form in the future. More info [here](https://ror.readme.io/docs/create-affiliation-selection-dropdowntypeahead-widgets)
+
+**Topic**
+
+The research field. Ideally, fine grained terms with hierarchical relationships. This would also be relevant for projects.
+
+* **Reason:** User persona + user feedback
+* **Mapping:** [about](https://schema.org/about)
+* **Value:** Maybe [DefinedTerm](https://schema.org/DefinedTerm)
+
+> Note: I am not sure what's the best way of implementing this. [Dataverse](https://demo.dataverse.org/dataverse/demo/search) uses a (short) flat list of terms, whereas [Swiss UBase](https://www.swissubase.ch/en/) uses a fine grained nested list. The closest thing I found is the ["Data Science Education Ontology (DSEO)"](https://fairsharing.org/FAIRsharing.7p0xdg). It has a [Domain](https://bioportal.bioontology.org/ontologies/DSEO/?p=classes&conceptid=http%3A%2F%2Fbigdatau.org%2Fdseo%23domain) class which subclasses owl#Thing.
+
+**Size**
+
+* **Reason:** user feedback
+* **Mapping:** [size](https://schema.org/size)
+* **Value:** [Text](https://schema.org/Text)
+* **Example:** `https://schema.org/size: "130MB"`
+
+> Note: It may not be possible to set it for external datasets.
+
+## Drawbacks
+
+> Why should we *not* do this? Please consider the impact on users,
+on the integration of this change with other existing and planned features etc.
+
+Proliferation of unnecessary fields may impact the performance and size of the knowledge graph, thus negatively affect user-experience.
+
+> There are tradeoffs to choosing any path, please attempt to identify them here.
+
+This may introduce problems with backwards compatibility, since the metadata structure would be modified.
+
+## Rationale and Alternatives
+
+> Why is this design the best in the space of possible designs?
+
+Adding generic information directly in the dataset metadata will make it easier for users to provide crucial context on their data. It is a first step towards encouraging them to fill in these fields.
+
+Currently, affiliation is a free-text value, which means that different values can refer to the same institution (e.g. UNIL and University of Lausanne), or a value may be ambiguous (UCD for University College Dublin or University of California, Davis).
+
+> What other designs have been considered and what is the rationale for not choosing them?
+
+Another option would be to keep these fields as custom annotations and provide tools to help users with data entry. The main issue with this solution is that the fields will not be available in the global KG and thus cannot be used in the search (e.g. to filter datasets in "Machine learning").
```

Comment:
Global KG should really get its act together and deal with custom metadata :stuck_out_tongue_closed_eyes: